src/components/cards/SightingsCard.jsx

```diff
@@ -179,7 +179,7 @@ export default function SightingsCard({
   );
 
   return (
-    <div>
+    <div style={{ marginTop: '0.5em', marginLeft: '0.5em' }}>
```

Comment:
so we actually dont use `em`s in our project, we use a 4px grid system. since `em` is the font size, 0.5em will usually work out to 7px which doesn't match the grid

Reply:
@brmscheiner Would you recommend doing my margins in pixels instead? Would that be wonky, depending on screen size? 

Perhaps this will be moot after the above gets resolved.